src/components/MasterItems.js

```diff
@@ -5,6 +5,19 @@ import _ from 'underscore';
 
 const MasterItemsSelect = createFactory(MasterItemsSelectComponent);
 
+function entriesByQuantity(entries) {
+  return entries.reduce((memo, entry) => {
+    let { quantity } = entry;
+    delete entry.quantity;
```

Comment:
Was sure how to handle this, but deleting to avoid sending quantity over to the server. If we weren't deleting, I think each entry in the server request would have the quantity attribute, i.e. [{foo, quantity: 2}, {foo, quantity: 2}].